content/desktop/hardened-desktop/enhanced-container-isolation/_index.md

```diff
@@ -92,16 +92,21 @@ To enable Enhanced Container Isolation as a developer:
 
 #### As an admin
 
+##### Prerequisite
+
 To enable Enhanced Container Isolation as an admin, you first need to [enforce
 sign-in](../../../security/for-admins/enforce-sign-in/_index.md). This is
 because the Enhanced Container Isolation feature requires a Docker Business
 subscription and therefore your Docker Desktop users must authenticate to your
-organization for this configuration to take effect. Enforcing sign-in ensures
-that your Docker Desktop developers always authenticate to your organization,
+organization for this configuration to take effect. 
+
+Enforcing sign-in ensures that your Docker Desktop developers always authenticate to your organization,
 even though they can authenticate without it and the feature will take effect.
 Enforcing sign-in guarantees the feature always takes effect.
```

Comment:
I think this paragraph reads strange. "your Docker Desktop developers" ?

Isn't it enough to just say that enforced sign-in is required? It feels like we can omit the bit about "they can authenticate without it".

```suggestion
Enforcing sign-in ensures that your Docker Desktop developers always authenticate to your organization.
```

Reply:
is enforced sign-in a prerquisite, or isn't it? "they can authenticate without it and the feature will take effect" seems to contradict that it is.